### ISSUE TITLE
Simplify branch: 'DNA replication, synthesis of RNA primer' ? 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#24385]
* 'GO:0006269 DNA replication, synthesis of RNA primer' -> 27 EXP 
* * '[GO:1902295](http://amigo.geneontology.org/amigo/term/GO:1902295#display-lineage-tab) synthesis of RNA primer involved in cell cycle DNA replication' ->  0 annotations
* * * GO:1902330 synthesis of RNA primer involved in bacterial-type DNA replication' ->  0 annotations
* * * 'GO:1902318 synthesis of RNA primer involved in nuclear cell cycle DNA replication' ->  0 annotations
* * * * GO:1902981 synthesis of RNA primer involved in mitotic DNA replication' -> 1 EXP pombase 
* * * * 'GO:1902981 synthesis of RNA primer involved in premeiotic DNA replication' ->  0 annotations

Do we need al the children of GO:0006269 DNA replication, synthesis of RNA primer? It seems to me and @colinlog that this is exactly the same process. 

The proposal is to obsolete all children with replace by GO:0006269 DNA replication, synthesis of RNA primer. 

@ValWood Is that OK for you ? 

Thanks, Pascale

**Comment by ValWood** [comment]
Yes, but in that case, the same should probably happen for all of the terms under DNA replication that mention meiosis, mitotic, or cell cycle . These were created historically so that cell cycle events could be positioned under mitotic cell cycle and meiotic cell cycle.

Personally, I think we should no longer do this in GO. I think we should only model "cell cycle control" under cell cycle, and not the actual processes they control (like DNA replication and chromosome segregation). Cell cycle controls many processes in addition to the macro events that were easily observed when cell cycle was defined. (mitochondrial processes,  vesicle-mediated transport, chromatin, transcription etc), so it seems impossible to model them all under cell cycle. Cell cycle researchers are more interested in the "regulation" or control of the transitions (which then allows the events to happen at the correct time), but  as it is now everything gets lumped together.
So yes I agree, but for complicated reasons!
